rviz_default_plugins/src/rviz_default_plugins/displays/marker/marker_common.cpp

```diff
@@ -50,7 +50,7 @@
 
 #include "rviz_default_plugins/displays/marker/markers/marker_factory.hpp"
 
-#include "../../ros_resource_retriever.hpp"
+#include "rviz_default_plugins/ros_resource_retriever.hpp"
```

Comment:
Was this part of the fix? I didn't see where this was being included in any other headers or anything. At least no new places. 

I made this in the source tree so this header wasn't installed and therefore was private as I don't think it's used outside, but I guess that might be useful. However, I didn't find that to be part of the fix.